platform-includes/getting-started-verify/javascript.nuxt.mdx

```diff
@@ -0,0 +1,14 @@
+```html {tabTitle:Vue} {filename:ErrorButton.vue}
+<script setup>
+  const triggerError = () => {
+  throw new Error("Nuxt Button Error");
+};
+</script>
+
+<template>
+  <button id="errorBtn" @click="triggerError">Trigger Error</button>
+</template>
+```
```

Comment:
l: Theoretically we could also show them how to capture an error on the server side. Not a high-prio task though, feel free to tackle later or ignore :)